core/common/src/main/java/alluxio/security/LoginUser.java

```diff
@@ -97,7 +97,7 @@ private static User login(Configuration conf) throws IOException {
       }
       return userSet.iterator().next();
     } catch (LoginException e) {
-      throw new IOException("Failed to login", e);
+      throw new IOException("Failed to login" + e);
```

Comment:
I intended to make the error message in LoginException explicit show up in IOException error msg so that end-user can see the reason for failed to login and proceed with fixes. Otherwise user has to dig into detailed error log to figure out why.

Per the javadoc of public IOException(String message, Throwable cause) :
Note that the detail message associated with cause is not automatically incorporated into this exception's detail message.

Added a space in between, thanks!